linkerd/tls/src/main/scala/io/buoyant/linkerd/tls/TlsUtils.scala

```diff
@@ -72,6 +87,19 @@ object TlsUtils {
     configFile
   }
 
+  def addDnsAltNamesInConfig(dir: File, names: Seq[String]): Unit = {
+    val configFile = new File(dir, "openssl.cfg")
+    val cw = new FileWriter(configFile, true)
+    cw.write(dnsAltNames(names))
+    cw.close()
+  }
+
+  def dnsAltNames(names: Seq[String]): String = {
+    val altNames = new Array[String](names.length)
```

Comment:
I think this can be written more simply as:

```
val altNames = names.zipWithIndex.map { case(name, i) => s"DNS.${i+1} = $name" }
altNames.mkString("\n")
```